tasks/sanity-label-check.yaml

```diff
@@ -7,28 +7,31 @@ spec:
   params:
     - name: sourceImage
       description: the fully qualified image name
-      default: quay.io/cvpops/test-index:v4.10
+  results:
+    - description: Test output
+      name: HACBS_TEST_OUTPUT
   steps:
     - name: inspect-image
```

Comment:
Should we remove this step and make use of the filed generated by inspection task https://github.com/redhat-appstudio/build-definitions/pull/42/files? Or update this task after https://github.com/redhat-appstudio/build-definitions/pull/42/files is merged?

Reply:
Ignore this message. This MR is just to update the test format and there is another task https://issues.redhat.com/browse/HACBS-289 for that update work.